mapview/view.py

```diff
@@ -206,7 +206,7 @@ def __init__(self, **kwargs):
     def insert_marker(self, marker, **kwargs):
         if self.order_marker_by_latitude:
             before = list(takewhile(
-                lambda (i, m): m.lat < marker.lat,
+                lambda i_m: i_m[1].lat < marker.lat,
```

Comment:
why not use `lambda i, m:  m.lat < marker.lat`? does that really no longer work? because in my tests it does appear to work:

```
$ python3
Python 3.7.2 (default, Jan 16 2019, 19:49:22) 
[GCC 8.2.1 20181215 (Red Hat 8.2.1-6)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> l = lambda x, y: (x, y)
>>> l(1, 2)
(1, 2)
>>>
```